tutorials/getting-started/schema-guide.md

```diff
@@ -344,6 +344,24 @@ Example:
 }
 ```
 
+
+The date picker UI can be configured by using the `pikadayOptions` configuration. We use [pikaday](https://github.com/dbushell/Pikaday#usage) and most configurations should work witout problems.
+
+Example with configuration of date picker:
+```javascript
+{
+  name: 'date',
+  label: 'Date',
+  type: 'date',
+  pikadayOptions: {
+    format: 'DD/MM/YYY',
```

Comment:
Example should match the default (and say so).

Reply:
Is YYY (three Y's) a thing? Not in the moment docs.

Reply:
No, YYY is a typo but this is actually pikaday format (which largely follows moment). I'll update that. Not sure with what you mean with "match the default"? You mean to put the example format as the default value? Because that for me is more confusing than an example of how this can be done in a different way.